api/controllers/environment_test.go

```diff
@@ -137,6 +139,53 @@ func TestReadEnvironment(t *testing.T) {
 	assert.Equal(t, expected, response)
 }
 
+func TestReadEnvironmentLogs(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockEnvironmentProvider := mock_provider.NewMockEnvironmentProvider(ctrl)
+	controller := NewEnvironmentController(mockEnvironmentProvider)
+
+	expected := []models.LogFile{
+		{
+			ContainerName: "alping",
```

Comment:
not that it really matters but I'm guessing you meant have `alpine`

Reply:
Yep, I will fix this.